appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 1.9.0.{build}
+environment:
+  TOKEN:
+    secure: YOUR-ENCRYPTED-CHOCO-APIKEY

```

Comment:
Have a look at https://github.com/StefanScherer/choco-docker-compose/blob/master/MAINTENANCE.md

Just pushing code only triggers build and test. As well as PRs are built and tedted and the contributor and you have a status icon after each commit in the PR. 

With `git tag 1.2.3 && git push --tags` You trigger another build and test plus the choco push deployment.